src/PSRule.Rules.Azure/rules/Azure.APIM.Rule.ps1

```diff
@@ -266,6 +266,14 @@ Rule 'Azure.APIM.MinAPIVersion' -Ref 'AZR-000321' -Type 'Microsoft.ApiManagement
     }
 } -Configure @{ AZURE_APIM_MIN_API_VERSION = '2021-08-01' }
 
+# Synopsis: API Management instances should use multi-region deployment to improve service availability.
+Rule 'Azure.APIM.MultiRegion' -Ref 'AZR-000340' -Type 'Microsoft.ApiManagement/service' -If { IsPremiumAPIM } -Tag @{ release = 'GA'; ruleSet = '2022_12'; } {
+    $multiRegion = @($TargetObject.properties.additionalLocations |
+        Where-Object { $_.location })
+    
+    $Assert.GreaterOrEqual($multiRegion, '.', 1).Reason($LocalizedData.APIMMultiRegion, $PSRule.TargetName)
```

Comment:
The updates here look good.

We can further simplify the rule with the following and I don't think we need to add `$PSRule.TargetName` since this information is already present in the PSRule output.

```suggestion
    $Assert.GreaterOrEqual($TargetObject, 'properties.additionalLocations', 1).Reason($LocalizedData.APIMMultiRegion, $PSRule.TargetName)
```